app/core/projection_engine_job.py

```
@@ -0,0 +1,64 @@
+from app.infrastructure.persistence.database import AsyncSessionLocal
+from app.infrastructure.persistence.repositories import (
+    EventRepository,
+    ProjectionRepository,
+    SubjectRepository,
+)
+from app.infrastructure.services.projection_engine import ProjectionEngine
+
+logger = logging.getLogger(__name__)
+
+
+async def run_projection_engine_job(app: FastAPI) -> None:
```

Ruff (0.15.5)

[warning] Unused function argument: `app`

(ARG001)